src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +119,205 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+		/// </summary>
+		/// <param name="obj">The object that modify the navigation bar visibility.</param>
+		/// <param name="value">true to set the navigation bar as visible; otherwise, false.</param>
 		public static void SetNavBarIsVisible(BindableObject obj, bool value) => obj.SetValue(NavBarIsVisibleProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetNavBarHasShadow']/Docs/*" />
+		/// <summary>
+		/// Gets if the navigation bar has a shadow. 
+		/// </summary>
+		/// <param name="obj">The object that modify if the navigation bar has a shadow.</param>
+		/// <returns>Returns if the navigation bar has a shadow.</returns>
 		public static bool GetNavBarHasShadow(BindableObject obj) => (bool)obj.GetValue(NavBarHasShadowProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetNavBarHasShadow']/Docs/*" />
+
+		/// <summary>
+		/// Controls whether the navigation bar has a shadow. 
+		/// By default the value of the property is <see langword="true"/> on Android, and <see langword="false"/> on other platforms.
+		/// </summary>
+		/// <param name="obj">The object that modify if the navigation bar has a shadow.</param>
+		/// <param name="value">Manages if the navigation bar has a shadow.</param>
 		public static void SetNavBarHasShadow(BindableObject obj, bool value) => obj.SetValue(NavBarHasShadowProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetSearchHandler']/Docs/*" />
+		/// <summary>
+		/// Gets the integrated search functionality in Shell.
+		/// </summary>
+		/// <param name="obj">The object that modify the Shell search functionality.</param>
+		/// <returns>The integrated search functionality in Shell.</returns>
 		public static SearchHandler GetSearchHandler(BindableObject obj) => (SearchHandler)obj.GetValue(SearchHandlerProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetSearchHandler']/Docs/*" />
+
+		/// <summary>
+		/// Enable the integrated search functionality in Shell.
+		/// Enabling it results in a search box being added at the top of the page.
+		/// </summary>
+		/// <param name="obj">The object that modify the Shell search functionality.</param>
+		/// <param name="handler">Defines the integrated search functionality in Shell.</param>
 		public static void SetSearchHandler(BindableObject obj, SearchHandler handler) => obj.SetValue(SearchHandlerProperty, handler);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetTabBarIsVisible']/Docs/*" />
+		/// <summary>
+		/// Gets the tabs visibility in Shell.
+		/// </summary>
+		/// <param name="obj">The object that modify the Shell tabs visibility.</param>
+		/// <returns>true if the tab bar is visible; otherwise, false.</returns>
```

Comment:
Cref true and false. I also noticed that in some places we refer to the tab bar as "tabbar" and in others as "tab bar". I think "tab bar" is better, but we should ensure this is consistent.